plugins/eddn.py

```diff
@@ -1657,6 +1745,18 @@ def journal_entry(  # noqa: C901, CCR001
         #     drop those events, or if the schema allows, send without those
         #     augmentations.
 
+        elif event_name == 'fsssignaldiscovered':
```

Comment:
Note to self: Will need to check if we have any signals queued up when a new Location/FSDJump/CarrierJump comes in, cross-check SystemAddress, send.

Keep in mind that in Horizons the Location/FSDJump comes *before* the new system's signals, in Odyssey it's after.